stable/rendertron/values.yaml

```diff
@@ -0,0 +1,42 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: ravishi/rendertron
```

Comment:
This image repo is manually built and only 4 pulls at the moment. Is there a better version of this image to use? Possibly one with automation or a pointer back to the config used to build it?

Reply:
Unfortunately, there is no better image/repo to use.  I've requested the rendertron project to publish official docker images in https://github.com/GoogleChrome/rendertron/issues/93.  Until that happens, I am building a pushing images myself.